src/theme/colors.ts

```diff
@@ -210,7 +217,7 @@ export const themes: Themes = {
       ...textTypes,
     },
     border: {
-      primary: colors.primary.gray_100,
+      primary: colors.primary.gray_50,
```

Comment:
Is this true for the entire design system?

Reply:
Yep, in every case I managed to find in app 1.0. Though with a different name, but same color value. Going over the new theming and new naming now in a separate update.